api/v1/trainingjob/trainingjob_types.go

```diff
@@ -96,6 +102,8 @@ type TrainingJobStatus struct {
 
 	//Full path to the training artifact (model)
 	ModelPath string `json:"modelPath,omitempty"`
+
+	DebugRuleEvaluationStatuses []commonv1.DebugRuleEvaluationStatus `json:"debugRuleEvaluationStatuses,omitempty"`
```

Comment:
done.

Reply:
@gautamkmr can we use proper capitalization on documentation like this? When we eventually publish docs, this will be customer facing. `s/debughookconfig/DebugHookConfig`.

I think we can make the sentence a bit more clear, something like 
```go
// Status of rule evaluation jobs, obtained from DebugRuleEvaluationStatuses.
// https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_DescribeTrainingJob.html#sagemaker-DescribeTrainingJob-response-DebugRuleEvaluationStatuses
```